plugins/mcreator-core/datalists/noteblockinstruments.yaml

```diff
@@ -0,0 +1,46 @@
+- harp:
+  readable_name: "Harp"
+- basedrum:
+  readable_name: "Bass Drum (Kick)"
+- snare:
+  readable_name: "Snare Drum"
+- hat:
+  readable_name: "Clicks and Sticks (Hi-hat)"
+- bass:
+  readable_name: "Bass (String Bass)"
+- flute:
+  readable_name: "Flute"
+- bell:
+  readable_name: "Bells (Glockenspiel)"
+- guitar:
+  readable_name: "Guitar"
+- chime:
+  readable_name: "Chimes"
+- xylophone:
+  readable_name: "Xylophone"
+- iron_xylophone:
+  readable_name: "Iron Xylophone (Vibraphone)"
+- cow_bell:
+  readable_name: "Cow Bell"
+- didgeridoo:
+  readable_name: "Didgeridoo"
+- bit:
+  readable_name: "\"Bit\" (Synthesizer)"
+- banjo:
+  readable_name: "Banjo"
+- pling:
+  readable_name: "\"Pling\" (Electric piano)"
+- zombie:
+  readable_name: "\"Zombie\""

```

Comment:
Why the quotation marks?

Reply:
I'll be honest, I took these names from MC Wiki and kept all the punctuation.

Reply:
I would remove those honestly as it will look a bit funky

Reply:
:heavy_check_mark: Fixed